src/fsharp/CompileOps.fs

```diff
@@ -2149,6 +2149,8 @@ type TcConfigBuilder =
       mutable internalTestSpanStackReferring: bool
 
       mutable noConditionalErasure: bool
+      
+      mutable pathMap: PathMap
```

Comment:
nit: the order here (`noConditionalErasure, pathMap`) doesn't match the order used below (`pathMap, noConditionalErasure`)